client/app/components/user/session.service.js

```diff
@@ -320,140 +328,149 @@ class SessionService {
   }
 
   getUserMenu(currentUser) {
-    return {
-      name: 'user',
-      title: currentUser.firstName,
-      iconClass: 'icon-user',
-      nichandle: currentUser.nichandle,
-      fullName: `${currentUser.firstName} ${currentUser.lastName}`,
-      subLinks: [
+    const useNewText = ['FR'].includes(currentUser.ovhSubsidiary);
+
+    return (useNewText
+      ? this.Navbar.buildMenuHeader(`
+        ${this.$translate.instant('common_menu_support_userAccount_1', { username: currentUser.firstName })}
+        <br>
+        ${this.$translate.instant('common_menu_support_userAccount_2')}
+      `)
+      : currentUser.firstName)
+      .then(title => ({
+        name: 'user',
+        title,
+        iconClass: 'icon-user',
+        nichandle: currentUser.nichandle,
+        fullName: `${currentUser.firstName} ${currentUser.lastName}`,
+        subLinks: [
         // My Account
-        {
-          name: 'user.account',
-          title: this.$translate.instant('global_account'),
-          url: '#/useraccount/infos',
-          click: () => this.trackUserMenuSection('my_account', 'account'),
-          subLinks: [{
-            title: this.$translate.instant('global_account_infos'),
+          {
+            name: 'user.account',
+            title: this.$translate.instant('global_account'),
             url: '#/useraccount/infos',
-          }, {
-            title: this.$translate.instant('global_account_security'),
-            url: '#/useraccount/security',
+            click: () => this.trackUserMenuSection('my_account', 'account'),
+            subLinks: [{
+              title: this.$translate.instant('global_account_infos'),
+              url: '#/useraccount/infos',
```

Comment:
We should target dedicated manager instead of `useraccount` which redirect to dedicated.